app/src/main/java/org/oppia/app/player/state/StatePlayerRecyclerViewAssembler.kt

```diff
@@ -128,6 +131,7 @@ class StatePlayerRecyclerViewAssembler private constructor(
    * answers header to help locate the items in the recycler view (when present).
    */
   private val previousAnswerViewModels: MutableList<StateItemViewModel> = mutableListOf()
+
```

Comment:
please run the ktlint command and fix the issues coming on different lines

Reply:
done

Reply:
done